### DISCUSSION TITLE
Meerkat REPLY form activation

### DISCUSSION THREAD

**Discussion by CurtisZenFu** [statamic/cms#7123] [Q&A]
Total newb on Git. Finally joined because I need answers and hope someone can help as there is almost nothing on Google-Fu for this.

1) Following the docs for Meerkat to the letter. Even setup the "Cool Writings" starter kit to be 100% exact.

2) I can a) get the submit form working (Integrate Meerkat) and b) get the comment thread loading (The Comment Thread) but...

3) For the life of me absolutely nothing happens with the stated "Reply Forms" instructions.
https://meerkatcomments.com/docs/allowing-visitors-to-reply-to-each-other

All it states is add an |a href="#"| link (where???) with the required passed data fields... but nothing happens. I click the link and it goes exactly where href points to > the top of the page with a "#" is appended to my URL.

Here is my code wrapped in a vanilla div.

`

        {{ meerkat:responses }}

        {{ comments }}
        <ul>
            <li>
                <div>
                    <p>{{ author.name }}:</p>
                    <p>{{ content }}</p>
                    <hr>
                    <div>
                        <a href="#" data-meerkat-form="reply" data-meerkat-reply-to="{{ id }}">Reply</a>
                    </div>    
                </div>
            </li>
        </ul>
        {{ /comments }}

        {{ /meerkat:responses }}

`
 
Barebones. CSS comes later. I just want to get that reply form to load.

Yes... {{ meerkat:replies-to }} is loaded in the HEAD and shows as such in browser dev tools and I can "load" the min.js file in my browser as well so clearly the server has access to the "replies-to" javascript that _should_ load the form right beneath the comment thread render.

What am I doing wrong?

Thank you in advance,

Total Dev Newb.

**Comment by duncanmcclean** [comment]
Sorry, this repository is only for Discussions relating to Statamic Core.

For third party addons, like Meerkat, you should open an issue/discussion on the addon's own GitHub repository: https://github.com/Stillat/meerkat-statamic3/issues

**Reply by CurtisZenFu** [reply]
As I said... newb... thank you.

**Reply by duncanmcclean** [reply]
No worries - have a great rest of your day! 😄

**Reply by CurtisZenFu** [reply]
Already had my question answered. You guys rock!
